src/nvs.rs

```diff
@@ -736,6 +767,72 @@ impl RawHandle for EspNvs<NvsEncrypted> {
         self.1
     }
 }
+impl RawHandle for EspNvs<NvsDefault> {
+    type Handle = nvs_handle_t;
+
+    fn handle(&self) -> Self::Handle {
+        self.1
+    }
+}
+
+#[cfg(esp_idf_version_at_least_5_2_5)]
```

Comment:
I don't think this is the minimum version where `nvs_entry_find_in_handle` appears. Did you actually check this in the ESP-IDF source code?